ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java

```diff
@@ -184,16 +184,18 @@ private static void setTransformProperty(View view, ReadableArray transforms) {
 
     if (perspectiveArray.length > PERSPECTIVE_ARRAY_INVERTED_CAMERA_DISTANCE_INDEX) {
       float invertedCameraDistance = (float) perspectiveArray[PERSPECTIVE_ARRAY_INVERTED_CAMERA_DISTANCE_INDEX];
-      if (invertedCameraDistance < 0) {
-        float cameraDistance = -1 / invertedCameraDistance;
-        float scale = DisplayMetricsHolder.getScreenDisplayMetrics().density;
+      if (invertedCameraDistance == 0) {
```

Comment:
In which cases `invertedCameraDistance` is `0` and why?
Does `0` make any logical sense  as `invertedCameraDistance` value?